MetadataExtractor/Formats/Exif/makernotes/NikonType2MakernoteDescriptor.cs

```diff
@@ -98,7 +100,17 @@ public override string GetDescription(int tagType)
         [CanBeNull]
         public string GetPowerUpTimeDescription()
         {
-            return GetEpochTimeDescription(NikonType2MakernoteDirectory.TagPowerUpTime);
+            // this is generally a byte[] of length 8 directly representing a date and time.
+            // the format is : first 2 bytes together are the year, and then each byte after
+            //                 is month, day, hour, minute, second with the eighth byte unused
+            // e.g., 2011:04:25 01:54:58
+
+            var o = Directory.GetByteArray(NikonType2MakernoteDirectory.TagPowerUpTime);
+            if (o == null)
+                return null;
+
+            ushort year = ByteConvert.FromBigEndianToNative(BitConverter.ToUInt16(o, 0));
+            return string.Format($"{year}:{o[2]:D2}:{o[3]:D2} {o[4]:D2}:{o[5]:D2}:{o[6]:D2}");
```

Comment:
I was thinking this is a pretty weird date format, but it's consistent with other dates the library returns via descriptors (it comes from the fact that Exif was the first format implemented). Do you think it'd make sense for descriptor methods to return culture-specific date strings? If so I'll file a follow-up issue.